### PR TITLE
read release state from group config

### PR DESCRIPTION
**Pull request by locriandev**
Ref. https://issues.redhat.com/browse/ART-6729

Follows https://github.com/openshift-eng/aos-cd-jobs/pull/3713

### CODE REVIEW

File: jobs/build/promote/Jenkinsfile [hunk 1]
```diff
@@ -307,7 +307,10 @@ node {
 
     if (major > 3 && ga_release) {
         def next_minor = "${major}.${minor + 1}"
-        if (!(commonlib.ocpReleaseState[next_minor] && commonlib.ocpReleaseState[next_minor]["release"])) {
+        def out = buildlib.doozer("--group=openshift-${next_minor} config:read-group --yaml release_state",
```
Comment (joepvd) [inline jobs/build/promote/Jenkinsfile:310]: promote is deprecated. better not touch it

File: jobs/build/multi_promote/Jenkinsfile [hunk 2]
```diff
@@ -129,11 +128,15 @@ node {
         currentBuild.displayName = release_name
     }
 
-    def arches = commonlib.ocpReleaseState["${major}.${minor}"]?.release
+    def major_minor = "${major}.${minor}"
+    def out = buildlib.doozer("--group=openshift-${major_minor} config:read-group --yaml release_state",
```
Comment (joepvd) [inline jobs/build/multi_promote/Jenkinsfile:132]: also deprecated. better not touch

**Review by joepvd — APPROVED** [review]
two suggestions, but is good as is

/lgtm

File: jobs/build/multi_promote/Jenkinsfile (hunk 2 above)
Comment (locriandev) [reply jobs/build/multi_promote/Jenkinsfile]: Removed with https://github.com/openshift-eng/aos-cd-jobs/pull/3722

File: jobs/build/promote/Jenkinsfile (hunk 1 above)
Comment (locriandev) [reply jobs/build/promote/Jenkinsfile]: Removed with https://github.com/openshift-eng/aos-cd-jobs/pull/3722

**Pull request merged**